package.json

```diff
@@ -100,11 +102,11 @@
     "sass-loader": "^10.1.1",
     "server-destroy": "^1.0.1",
     "sluggo": "^0.3.0",
-    "stylelint": "^13.13.0",
-    "stylelint-config-standard": "^20.0.0",
-    "stylelint-declaration-strict-value": "^1.7.12",
-    "stylelint-order": "^4.1.0",
-    "stylelint-webpack-plugin": "^2.1.1",
+    "stylelint": "^14.6.1",
+    "stylelint-config-standard": "^25.0.0",
+    "stylelint-declaration-strict-value": "^1.8.0",
+    "stylelint-order": "^5.0.0",
+    "stylelint-webpack-plugin": "^3.2.0",
```

Comment:
`stylelint-*` dependencies must be updated as well, to match the `stylelint@14` version in their peer dependencies